sky/optimizer.py

```diff
@@ -969,13 +977,13 @@ def _fill_in_launchable_resources(
     blocked_resources: Optional[Iterable[resources_lib.Resources]],
     try_fix_with_sky_check: bool = True,
 ) -> Tuple[Dict[resources_lib.Resources, List[resources_lib.Resources]],
-           _PerCloudCandidates]:
```

Comment:
nit: Can we add a docstr for the return values, considering the long return tpule?

Reply:
Done.